guided-charts/aws-hyperpod/values.yaml

```diff
@@ -25,3 +25,53 @@ storageClass:
     parameters:
       type: gp3
       fsType: ext4
+
+# Authmiddleware configuration
+authmiddleware:
+  # Whether to deploy the authmiddleware component
+  enabled: true
+  # Number of replicas
+  replicas: 1
+  # Authmiddleware image settings
+  imagePullPolicy: Always
+  repository: docker.io
+  imageName: jupyter-k8s-auth
+  imageTag: latest
+  # Resource limits and requests
+  resources:
+    limits:
+      cpu: 200m
+      memory: 128Mi
+    requests:
+      cpu: 200m
+      memory: 128Mi
+  # JWT configuration
+  # JWT validation type: "hmac" or "kms"
+  jwtValidationType: "kms"
+  jwtIssuer: "jupyter-k8s-auth"
+  jwtAudience: "workspace-users"
+  jwtExpiration: "1h"
+  jwtRefreshWindow: "15m"
+  jwtRefreshHorizon: "12h"
```

Comment:
callout: #98 is adding `jwtRefreshEnable`, which this chart should set to `false` (it defaults to `true`).

I can add it to #98 if this PR goes in first.